container/metadata.csv

```diff
@@ -17,7 +17,7 @@ container.memory.limit,gauge,,byte,,The container memory limit,0,container,mem_l
 container.memory.major_page_faults,count,,,,Number of major page faults incurred,0,container,pgmajfault,
 container.memory.oom_events,gauge,,,,The number of OOM events triggered by the container,0,container,mem_oom_events,
 container.memory.page_faults,count,,,,Total number of page faults incurred,0,container,pgfault,
-container.memory.rss,gauge,,byte,,The container RSS usage,0,container,mem_rss,
+container.memory.rss,gauge,,byte,,The 'anon' memory used by the container,0,container,mem_rss,
```

Comment:
We can perhaps improve more with the Kernel description:
```
Amount of memory used in anonymous mappings such as brk(), sbrk(), and mmap(MAP_ANONYMOUS)
```

Reply:
I was hesitant to include more specific linux concepts as this appears to be cross-platform, and I don't know what this reports on windows.

Reply:
It's a Linux-only metric.